pkg/coscheduling/core/core.go

```diff
@@ -269,13 +269,13 @@ func (pgMgr *PodGroupManager) calculateAssignedPods(podGroupName, namespace stri
 	return count
 }
 
-func (pgMgr *PodGroupManager) CheckClusterResource(nodeList []*framework.NodeInfo, resourceRequest corev1.ResourceList) error {
+func (pgMgr *PodGroupManager) CheckClusterResource(nodeList []*framework.NodeInfo, resourceRequest corev1.ResourceList, desiredPodGroupName string) error {
```

Comment:
Please add a comment for this function.

Off the topic: should we add the PG into permitted cache in L121?

Reply:
> Please add a comment for this function.
> 
> Off the topic: should we add the PG into permitted cache in L121?

IMO, add or not is ok , permitted cache is used for avoiding computing  resources repeatedly